views/admin/index.php

```diff
@@ -99,7 +101,16 @@
         ],
         [
             'class' => 'yii\grid\ActionColumn',
-            'template' => '{update} {delete}',
+            'template' => '{resend_password} {update} {delete}',
+            'buttons' => [
+                'resend_password' => function ($url, $model, $key) {
+                    if (!$model->isAdmin)
```

Comment:
`if`, `else`, `elseif` [must have](http://www.php-fig.org/psr/psr-2/#if-elseif-else) opening and closing braces